Jenkinsfile_CNP

```diff
@@ -6,4 +6,44 @@ def type = "java"
 def product = "wa"
 def component = "task-monitor"
 
-withPipeline(type, product, component) {}
+def secrets = [
+  's2s-${env}': [
+    secret('microservicekey-wa-task-monitor', 'S2S_SECRET_TASK_MONITOR'),
+  ]
+]
+
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
+  [$class     : 'AzureKeyVaultSecret',
+   secretType : 'Secret',
+   name       : secretName,
+   version    : '',
+   envVariable: envVar
+  ]
+}
+
+withPipeline(type, product, component) {
+
+  disableLegacyDeployment()
+
+  env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+  env.CAMUNDA_URL = "http://camunda-api-aat.service.core-compute-aat.internal/engine-rest"
+  env.TASK_CONFIGURATION_SERVICE_URL = "http://wa-task-configuration-api-aat.service.core-compute-aat.internal"
+
+  loadVaultSecrets(secrets)
+
+  after('test') {
+    steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/tests/**/*'
+    steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/tests/integration/**/*'
```

Comment:
There is an existing tech debt to ensure we generate reports for integration tests as well. Please check this will be generated for task-monitor app as we have integration test

Reply:
sure, do you know the jira number?

Reply:
https://tools.hmcts.net/jira/login.jsp?permissionViolation=true&os_destination=%2Fbrowse%2FRWA-515&page_caps=&user_role=

Reply:
done